### PR TITLE
Update PR Template by making comments to be comments

### PR DESCRIPTION
**Pull request by nammn**
PR FOR DISCUSSION

### Checklist

 - [ ] Checked if it requires a [pylibjuju](https://github.com/juju/python-libjuju) change?
 - [ ] Added [integration tests](https://github.com/juju/juju/tree/develop/tests) for the PR?
 - [ ] Added or updated [doc.go](https://discourse.jujucharms.com/t/readme-in-packages/451) related to packages changed?
 - [ ] Do comments answer the question of why design decisions were made?

----

## Description of change

- Usually, if I open a PR I read the comments and try to work on each point. After done I delete the corresponding "comment".
- As the information is meant not be shown in the PR itself IMO we could change them to be comments and only seeable for the person opening/editing the PR

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -11,20 +11,21 @@
...
 ## Documentation changes
 
-*Please replace with any notes about how it affects current user workflow? CLI? API?*
+<!--- *Please replace with how we can verify that the change works?* 
+ *Please replace with any notes about how it affects current user workflow? CLI? API?* -->
 
 ## Bug reference
 
-*Please add a link to any bugs that this change is related to.*
+<!--*Please add a link to any bugs that this change is related to.* -->
```
Comment (manadart) [inline PULL_REQUEST_TEMPLATE.md]: Missed a dash here. 

I would remove the the other markdown parts from the comments too, "*" and "##".

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -11,20 +11,21 @@
...
+<!--- *Please replace with how we can verify that the change works?* -->
 
 ```sh
 QA steps here
 ```
 
 ## Documentation changes
 
-*Please replace with any notes about how it affects current user workflow? CLI? API?*
+<!--- *Please replace with how we can verify that the change works?*
```
Comment (manadart) [inline PULL_REQUEST_TEMPLATE.md]: Copy/paste?

**Comment by nammn** [comment]
@manadart ah thanks for the rev! And what do you think of this in general?

**Comment by manadart** [comment]
In principle, I'm good with it. Though I'd be interested in what the merge bot does with the description to make the patch commit message.

If we remove the motivation to delete the template guide comments, so they end up as noise in commit message text?

**Comment by nammn** [comment]
Will look into this later on a ci day with the mergebot. Hence closing for now

**Pull request closed without merging**